api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -127,6 +127,33 @@ public void savePatientProgram_shouldUpdatePatientProgram() {
 		
 	}
 
+	/**
+	 * @see ProgramWorkflowService#voidPatientProgram(PatientProgram, String)
+	 */
+	@Test
+	public  void voidPatientProgram_shouldVoidPatientProgram() {
+		PatientProgram patientProgram = Context.getProgramWorkflowService().getPatientProgram(1);
```

Comment:
can you please setup the object you need for the test without using the DB, so directly within the test so its easy to see what specific test case you are testing. When using the DB this information is hidden and the test is hard to read.

in your case its `PatientProgram`